scripts/SMD_chip_package_rlc-etc/size_definitions/size_inductor.yaml

```diff
@@ -1,4 +1,18 @@
-SMD_1008:
+L_0805:
```

Comment:
I don't see L_0603 here. So The one from size_default_chip is used.
Is that by design? Or should we also include a explicit 0603 inductor?

Reply:
See page 80 of the IPC-SM-782 doc linked: there is no 0603 body size for chip inductors. So it is intentional. I added a more explicit comment in SMD_packages.yaml about this.